cupy_backends/cuda/libs/cusparse.pyx

```diff
@@ -1572,45 +1572,46 @@ cpdef intptr_t create() except? 0:
     return <intptr_t>handle
 
 
-cpdef size_t createMatDescr():
+cpdef size_t createMatDescr() except? -1:
```

Comment:
🐞 For functions returning C-types and returning exceptions, `except` must be declared.